tests/relay/test_sorgenti_squadre.py

```
@@ -0,0 +1,497 @@
+                 {'alias': {str(ju): 'X'}}),
+                ('PATCH', f'/api/me/sorgenti-squadre/{sid}', {'nome': 'rubata'}),
+                ('DELETE', f'/api/me/sorgenti-squadre/{sid}', None),
+        ):
+            stato, risposta, _ = _chiama(servizio, metodo, percorso,
+                                         corpo=corpo, cookie=cookie_b)
+            assert stato == 404, f'{metodo} {percorso}: atteso 404, avuto {stato} {risposta[:120]!r}'
+        # e per A e' ancora tutto al suo posto
+        assert _leggi_alias(servizio, cookie_a, cid, sid) == {'Juventus': ''}
+    finally:
+        _chiama(servizio, 'DELETE', f'/api/me/sorgenti-squadre/{sid}', cookie=cookie_a)
+        _chiama(servizio, 'DELETE', f'/api/me/sports/privato', cookie=cookie_a)
```

Ruff (0.16.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)